### PR TITLE
Feature/homepage ux changes

### PR DESCRIPTION
**Pull request by seanjessen**

### CODE REVIEW

File: src/components/Search/store.ts [hunk 1]
```diff
@@ -18,13 +23,40 @@ class SearchStore {
...
+    });
+  };
+
+  @action geolocate = async () => {
+    try {
+      const geolocation = await axios.get(
+        `https://maps.googleapis.com/maps/api/geocode/json?latlng=${this.locationCoords.lat},${this.locationCoords.lon}&result_type=postal_code&key=${process.env.REACT_APP_GOOGLE_API_KEY}`
+      );
+
+      console.log(geolocation);
```
Comment (matthew-inamdar) [inline src/components/Search/store.ts:52]: Can we remove this log?

File: src/components/Search/store.ts [hunk 2]
```diff
@@ -18,13 +23,40 @@ class SearchStore {
...
+
+      this.location = get(geolocation, 'data.results[0].formatted_address', {});
+    } catch (e) {
+      console.error(e);
```
Comment (matthew-inamdar) [inline src/components/Search/store.ts:56]: What happens if the geolocation fails?

**Review by matthew-inamdar — CHANGES_REQUESTED** [review]
Looks good 👍 

Left one change request, and a question.

File: src/components/Search/store.ts (hunk 1 above)
Comment (seanjessen) [reply src/components/Search/store.ts]: Removed log

File: src/components/Search/store.ts (hunk 2 above)
Comment (seanjessen) [reply src/components/Search/store.ts]: Added alert here

**Review by matthew-inamdar — APPROVED** [review]
LGTM 👍

**Pull request merged**